### ISSUE TITLE
Isolated builds via pip aren't operating properly when building fron non-local directory.

### ISSUE THREAD

**Issue by whardier** [EducationalTestingService/match#26]
The "requirements" path needs to be resolved and relative to the setup.py.  Typically use __file__ to handle this - but ultimately should just switch to using setuptools/pyproject.toml

Since there is no binary component it would be wise to produce wheel files as well when publishing to pypi.

```
# INFO: pip is looking at multiple versions of match to determine which version is compatible with other requirements. This could take a while.
# Collecting match~=0.3
#     Using cached match-0.3.1.tar.gz (8.7 kB)
#     Preparing metadata (setup.py): started
#     Preparing metadata (setup.py): finished with status 'error'
#     error: subprocess-exited-with-error

#     × python setup.py egg_info did not run successfully.
#     │ exit code: 1
#     ╰─> [8 lines of output]
#         Traceback (most recent call last):
#         File "<string>", line 2, in <module>
#         File "<pip-setuptools-caller>", line 34, in <module>
#         File "/private/var/folders/42/5m231dl54c5c2g_frb8l7z_w0000gp/T/pip-resolve-zsn2i4rv/match_e3bfb10a2efe459f92225bad0d9be0bf/setup.py", line 37, in <module>
#             install_requires=requirements()
#         File "/private/var/folders/42/5m231dl54c5c2g_frb8l7z_w0000gp/T/pip-resolve-zsn2i4rv/match_e3bfb10a2efe459f92225bad0d9be0bf/setup.py", line 12, in requirements
#             with open('requirements.txt', 'r') as fh:
#         FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
#         [end of output]
```

**Comment by desilinguist** [comment]
This was fixed with [v0.3.2](https://github.com/EducationalTestingService/match/releases/tag/v0.3.2).

**Comment by whardier** [comment]
Augh.. sorry.  For some reason the backtracking resolver wanted to see if this was more appropriate and it attempted to install 0.3.1
